purchase_order_line_description/models/purchase_order_line.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2015 Alex Comba - Agile Business Group
+# Copyright 2017 Tecnativa - vicent.cubells@tecnativa.com
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = "purchase.order.line"
+
+    @api.onchange("product_id")
+    def onchange_product_id(self):
```

Comment:
As stated in coding guideline
```suggestion
    def _onchange_product_id(self):
```

Reply:
It can be kept as it is as has the same name as used in their inheritance.